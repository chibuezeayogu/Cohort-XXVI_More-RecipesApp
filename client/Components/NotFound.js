import React from 'react';
import { Row, Input, Icon} from 'react-materialize';


class NotFound extends React.Component{
    render() {
        return (
            <div className="container"> 
                <h1>Page Not Found</h1>
            </div>
        );
    }
}

export default NotFound;