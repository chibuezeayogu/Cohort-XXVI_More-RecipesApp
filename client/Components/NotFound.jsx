import React from 'react';
import UserMenu from './Header/UserMenu.jsx';
import Footer from './Footer/Footer.jsx';

/**
* The page the user gets when an invalid route is entered
* in the address bar.
*
* @description
*
* @method
*
* @returns {undifined} - JSX element
*/
const NotFound = () => (
  <div className="body">
    <UserMenu />
    <div className="main">
      <div className="container">
        <h1 className="center" style={{ marginTop: 200 }}>Page Not Found</h1>
      </div>
    </div>
    <Footer />
  </div>
);

export default NotFound;
