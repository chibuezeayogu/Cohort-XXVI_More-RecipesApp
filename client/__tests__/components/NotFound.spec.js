import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import NotFound from '../../Components/NotFound';

describe('<NotFound />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
