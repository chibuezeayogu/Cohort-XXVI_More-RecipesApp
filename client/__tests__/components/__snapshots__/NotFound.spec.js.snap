// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  "node": <div
    className="body"
  >
    <withRouter(Connect(UserMenu)) />
    <div
      className="main"
    >
      <div
        className="container"
      >
        <h1
          className="center"
          style={
            Object {
              "marginTop": 200,
            }
          }
        >
          Page Not Found
        </h1>
      </div>
    </div>
    <Footer />
  </div>,
  "nodes": Array [
    <div
      className="body"
    >
      <withRouter(Connect(UserMenu)) />
      <div
        className="main"
      >
        <div
          className="container"
        >
          <h1
            className="center"
            style={
              Object {
                "marginTop": 200,
              }
            }
          >
            Page Not Found
          </h1>
        </div>
      </div>
      <Footer />
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <NotFound />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="body"
        >
          <withRouter(Connect(UserMenu)) />
          <div
            className="main"
          >
            <div
              className="container"
            >
              <h1
                className="center"
                style={
                  Object {
                    "marginTop": 200,
                  }
                }
              >
                Page Not Found
              </h1>
            </div>
          </div>
          <Footer />
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="body"
        >
          <withRouter(Connect(UserMenu)) />
          <div
            className="main"
          >
            <div
              className="container"
            >
              <h1
                className="center"
                style={
                  Object {
                    "marginTop": 200,
                  }
                }
              >
                Page Not Found
              </h1>
            </div>
          </div>
          <Footer />
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <NotFound />,
}
`;
